signer/client/signer_trust_test.go

```diff
@@ -114,6 +114,18 @@ func TestGetPrivateKeyIfNoKey(t *testing.T) {
 	assert.Nil(t, privKey)
 }
 
+func TestCreatePrivateKey(t *testing.T) {
+	signer := setUpSigner(t, trustmanager.NewKeyMemoryStore(ret))
+	key, err := signer.Create(data.CanonicalSnapshotRole, "docker.com/notary", data.ECDSAKey)
+	assert.NoError(t, err)
+	assert.NotNil(t, key)
+	retrievedKey := signer.GetKey(key.ID())
+	assert.NotNil(t, retrievedKey)
+	assert.Equal(t, key.Public(), retrievedKey.Public())
+	assert.Equal(t, key.ID(), retrievedKey.ID())
+	assert.Equal(t, key.Algorithm(), retrievedKey.Algorithm())
```

Comment:
Oops, sorry, this is newish.  Do we want the expected value to be `data.ECDSAKey` here specifically, to ensure that the right algorithm got passed?  (Maybe also assertion like `assert.Equal(t, data.ECDSAKey, key.Algorithm())` up above where `retrieveKey` is assigned?)